google/cloud/bigquery/table.py

```diff
@@ -2142,7 +2142,7 @@ def __hash__(self):
         return hash(self._key())
 
     def __repr__(self):
-        key_vals = ["{}={}".format(key, val) for key, val in self._key()]
+        key_vals = ["{}={}".format(key, val.__repr__()) for key, val in self._key()]
```

Comment:
```suggestion
        key_vals = ["{}={}".format(key, repr(val)) for key, val in self._key()]
```
(nit)
A bit more readable this way, IMO.

Reply:
Yes I agree!